tools/Sendmail_relay/mailwatch-sendmail-relay

```diff
@@ -23,7 +23,7 @@
 # Author: @stefaweb, 14/06/2015
 
 NAME="mailwatch-sendmail-relay"
-DAEMON="/usr/local/bin/sendmail_relay.php"
+DAEMON="/usr/local/bin/mailwatch-sendmail-relay.php"
```

Comment:
Could you place this patch in a separate PR?
This a nothing to do with color chart.